.travis.yml

```diff
@@ -1,6 +1,11 @@
 language: c
 os: [linux, osx]
 osx_image: mavericks
+sudo: false
+addons:
+  apt:
+    packages:
+    - gawk
```

Comment:
This appears to be necessary for some reason at this point but may be able to be removed later.